vault/login_mfa.go

```diff
@@ -589,13 +589,10 @@ func (i *IdentityStore) handleLoginMFAAdminDestroyUpdate(ctx context.Context, re
 // loadMFAMethodConfigs loads MFA method configs for login MFA
 func (b *LoginMFABackend) loadMFAMethodConfigs(ctx context.Context, ns *namespace.Namespace) error {
 	b.mfaLogger.Trace("loading login MFA configurations")
-	barrierView, err := b.Core.barrierViewForNamespace(ns.ID)
-	if err != nil {
-		return fmt.Errorf("error getting namespace view, namespaceid %s, error %w", ns.ID, err)
-	}
+	barrierView := NamespaceView(b.Core.systemBarrierView, ns)
```

Comment:
This will give you `/sys/namespaces/<uuid>`, I assume you want `/namespaces/<uuid>/sys`.

```suggestion
	barrierView := NamespaceView(b.Core.barrier, ns).SubView(systemBarrierPrefix)
```